### ISSUE TITLE
Add access capabilities

### ISSUE THREAD

**Issue by gbrewste** [moodleuulm/moodle-local_profilecohort#16]
Adding capabilities to the plugin would be great. Since there are capabilities to manage cohorts themselves without having to be an admin, adding them to this plugin would make sense. I think two would make the most sense: 
* Manage rules - capability to add, change, delete rules
* View rules - capability to view the rules for particular cohort.

I like this plugin, especially being able to reproduce functionality found in Totara

**Comment by abias** [comment]
> Adding capabilities to the plugin would be great. Since there are capabilities to manage cohorts themselves without having to be an admin, adding them to this plugin would make sense.

I agree with this request, however I have to tell you that we don't have a local need for this feature here and thus don't have the resources to add it currently.

Please don't hesitate to create a pull request and we will evaluate it for integration.

> I like this plugin, especially being able to reproduce functionality found in Totara

Thank you, the mechanism in Totara to fill audiences was exactly what we had in mind when we created this plugin. 
Looking at Moodle workplace (https://moodle.com/workplace/), we see a similar functionality there, too. But nobody knows yet when / if this will be backported to Moodle core.

**Comment by jtsafran** [comment]
Hi all!  I see there has been a pull request and some checks that have passed.  Is this code going to be merged in?

**Comment by vishalkishor28** [comment]
Since there are capabilities to manage cohorts themselves without having to be an admin, adding them to this plugin would make sense. I think two would make the most sense:

Manage rules - capability to add, change, delete rules
View rules - capability to view the rules for particular cohort.
